wiki/Usage_permissions/en.md

```diff
@@ -0,0 +1,427 @@
+| wotoha                           ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/178)  |
+| Xanthochroid                     ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/152)  |
+| xi                               ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/727)  |
+| yaseta                           ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/342)  |
+| Yokomin                          ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/355)  |
+| Yooh                             ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/218)  |
+| YUC'e                            ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/372)  |
+| yuki.                            ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/4)    |
+| yukitani                         ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/167)  |
+| Yunosuke                         ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/244)  |
+| Yuyoyuppe / DJ'TEKINA//SOMETHING ![](FA) | 🟠 Allowed, with exceptions         | Tracks from or themed around Touhou should not be uploaded. | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/132)  |
```

Comment:
given the purpose of the page, I think the exception should be written in absolute terms rather than "should". something like "obtain the artist's permission for tracks themed around touhou" if that's what they were going for

Reply:
If the artist wanted users to obtain permission, then I think that would be written. This sounds like a hard ruling (@Ephemeralis will have to confirm).

Also "should" is quite an absolute term:

![CleanShot 2023-10-30 at 03 01 22](https://github.com/ppy/osu-wiki/assets/191335/5d71fa62-b2ae-4941-bd54-3dc3e0080391)

Reply:
It depends on the artist themselves, do they hard rule disallows non-FA (must) or asked first before uploading (should)? Either case is fine depending on the degree. We should confirm tk each artists whether explicit requests needed or disallowed at all.

Reply:
That's not how this page works. It is opt in and already lists artists that want to be asked explicitly.